src/styles/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product {
+  width: 200px;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
```

Comment:
move this rule to `*` selector

Reply:
There is another information in rules: 

> Don't use * selector (it impacts performance)
Set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for browser.